templates/search/blocks/search_results_tag_frequency.html

```diff
@@ -2,14 +2,14 @@
 
 <div class="tag-frequency">
     <div class="tag-frequency__tags">
-        <p class="tag-frequency__title">Filter by:</p>
+        <p class="tag-frequency__title">Refine by category:</p>
         <ul class="tag-frequency__tags-list">
 
           
                 <a href="{% tag_type_url tag_type=tag_type.LOCATION|upper %}#tagTop" class="ohos-tag ohos-tag--location">
 
                     <span class="ohos-tag__inner">
-                        Location
+                        <label for="Category: Location">Location</label>
```

Comment:
Label is usually associated with an input field. 
Getting this browser message:
The label's for attribute doesn't match any element id. This might prevent the browser from correctly autofilling the form and accessibility tools from working correctly.
To fix this issue, make sure the label's for attribute references the correct id of a form field.

Reply:
Now changed to aria-label tags on the <a href> elements, this is consistent, for reference : https://www.aditus.io/aria/aria-label/